Build/windows/build.bat

```diff
@@ -10,12 +10,13 @@ set "build=y"
 if NOT -%1-==-- call :args %*
 if exist CMakeCache.txt del /f /s /q CMakeCache.txt 1>nul
 if exist CMakeFiles rmdir /s /q CMakeFiles 1>nul
-if "%buildtype%"=="" set "release"
+if "%buildtype%"=="" set "buildtype=release"
 if "%GENERATOR%"=="Visual Studio 16 2019" (
-    cmake ../.. -G"Visual Studio 16 2019" -A x64 -DCMAKE_INSTALL_PREFIX=%SYSTEMDRIVE%\svt-encoders -DCMAKE_CONFIGURATION_TYPES="%buildtype%" %cmake_eflags%
+    cmake ../.. -G"Visual Studio 16 2019" -A x64 -DCMAKE_INSTALL_PREFIX="%SYSTEMDRIVE%\svt-encoders" -DCMAKE_CONFIGURATION_TYPES="%buildtype%" %cmake_eflags%
+) else if NOT "%GENERATOR%"=="" (
+    cmake ../.. -G"%GENERATOR%" -DCMAKE_INSTALL_PREFIX="%SYSTEMDRIVE%\svt-encoders" -DCMAKE_CONFIGURATION_TYPES="%buildtype%" %cmake_eflags%
 ) else (
-    if NOT "%GENERATOR%"=="" set GENERATOR=-G"%GENERATOR%"
-    cmake ../.. %GENERATOR% -DCMAKE_INSTALL_PREFIX=%SYSTEMDRIVE%\svt-encoders -DCMAKE_CONFIGURATION_TYPES="%buildtype%" %cmake_eflags%
+    cmake ../.. -DCMAKE_INSTALL_PREFIX="%SYSTEMDRIVE%\svt-encoders" -DCMAKE_CONFIGURATION_TYPES="%buildtype%" %cmake_eflags%
```

Comment:
If GENERATOR is empty, build will fail. We should return error here, not let it continue.

Reply:
If GENERATOR is empty, cmake will choose the generator.

Run `cmake -G` and look for asterisk for default generator